docs/src/examples/components/Portal/Types/PortalExampleFocusTrapped.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Button, Header, Portal } from '@stardust-ui/react'
+
+class PortalExamplePortal extends React.Component {
+  render() {
+    return (
+      <Portal
+        trigger={
+          <Button
+            content={'Toggle portal'}
+            trapFocus={true}
```

Comment:
is this example correct? should the trapFocus be set on the trigger Button?

Reply:
you're right, it's not correct. Was testing with shorthand example. Thank you that you caught it!